api/provider/load_balancer.go

```diff
@@ -7,4 +7,5 @@ type LoadBalancerProvider interface {
 	Read(loadBalancerID string) (*models.LoadBalancer, error)
 	List() ([]models.LoadBalancerSummary, error)
 	Delete(loadBalancerID string) error
+	Update(req models.UpdateLoadBalancerRequest) (*models.LoadBalancer, error)
```

Comment:
Let's have updates return only an `error`. Given the job pattern, client's are going to call a `Read` after the job is completed anyways.